rest/blip_sync.go

```diff
@@ -80,10 +80,10 @@ func (h *handler) handleBLIPSync() error {
 	// Create a BLIP context:
 	blipContext := blip.NewContext()
 	blipContext.Logger = func(fmt string, params ...interface{}) {
-		base.LogTo("BLIP", fmt, params...)
+		base.LogTo("Sync", fmt, params...)
```

Comment:
This makes it tricky to output `Sync+` prefixed log messages as a result of `blipContext.LogMessages` callbacks and `Sync++` prefixed log messages as a result of  `blipContext.LogFrames` callbacks.

Reply:
The nicest way I can think of doing this without polluting the go-blip API with SG logging concepts, is being able to supply 3 logger callbacks.

Maybe removing the `LogMessages`/`LogFrames` bools, and use a nil callback func to mean disabled.

E.g: https://play.golang.org/p/Pz9sf7QyqLz

Reply:
@bbrks the plan we've ended up on (so far) was to have blip pass back an extra enum (normal | message | frame), and then the logger callback would use the appropriate log key, depending on the enum val